.github/workflows/docker/Dockerfile-compile-check-ubuntu18.04

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Shirong Hao <shirong@linux.alibaba.com>"
+
+RUN apt-get update && apt-get install -y autoconf gcc make wget git \
+    libseccomp-dev binutils-dev libprotoc-dev libprotobuf-c0-dev \
+    protobuf-c-compiler pkg-config libssl-dev openssl
+
+WORKDIR /root
+
+# install go
+RUN wget https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz && \
+    tar -zxvf go1.14.2.linux-amd64.tar.gz -C /usr/lib && \
+    rm -rf go1.14.2.linux-amd64.tar.gz
+
+# configure GOPATH and GOROOT
+ENV GOROOT       /usr/lib/go
+ENV GOPATH       /root/gopath
+ENV PATH         $PATH:$GOROOT/bin:$GOPATH/bin
```

Comment:
It's better to change `ENV PATH         $PATH:$GOROOT/bin:$GOPATH/bin` to `ENV PATH         $GOROOT/bin:$GOPATH/bin:$PATH`. Because there can many version `go` binary in the same system, we need set the `go` binary which located in `$GOROOT/bin` or `$GOPATH/bin` a higher priority than the `go` binary which located in other `$PATH`.

Reply:
ditto